packages/sit-onyx/src/i18n/locales/en-US.json

```
@@ -267,5 +267,11 @@
       "select": "Select"
     },
     "system": "System"
+  },
+  "itemsPerPage": {
+    "label": "Results per Page",
+    "select": {
+      "listLabel": "Select number of results per Page"
+    }
```

```suggestion
    "label": "Items per page",
    "select": {
      "listLabel": "Select number of items per page"
    }
```

Would prefer to keep the translations aligned with the component name :)